content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst

```diff
@@ -0,0 +1,37 @@
+=================
+Payment providers
+=================
+
+The default payment method for the eCommerce app is the **wire transfer**. Although not mandatory,
+it is strongly recommended to add additional payment providers for your customers. Odoo supports a
+multitude of online :doc:`payment providers <../../../../applications/finance/payment_providers>`
+for your website.
+
+.. seealso::
+   :doc:`../../../../applications/finance/payment_providers`
+
+Configuration
+-------------
+
+To setup payment providers on the eCommerce app, go to :menuselection:`Website --> Configuration -->
```

Comment:
```suggestion
To set up payment providers on the eCommerce app, go to :menuselection:`Website --> Configuration -->
```
[Setup is one word when it is a noun or an adjective. It is two words when it functions as a verb.](https://grammarist.com/spelling/set-up-vs-setup/)